extensions/parquet/table/src/main/java/io/deephaven/parquet/table/location/ParquetTableLocation.java

```diff
@@ -623,58 +621,50 @@ private Pair<Object, Object> getMinMax(final Statistics<?> statistics) {
         return null;
     }
 
-    /**
-     * Apply the range filter to the row groups and return the result.
-     */
-    @NotNull
-    private PushdownResult pushdownRangeFilter(
+    private WritableRowSet removeExclusions(
             final AbstractRangeFilter rf,
             final List<Integer> parquetIndices,
-            final PushdownResult result) {
-        final RowSetBuilderSequential maybeBuilder = RowSetFactory.builderSequential();
-        final MutableLong maybeCount = new MutableLong(0);
-
+            final RowSet maybeMatch) {
+        final RowSetBuilderSequential newMaybeMatch = RowSetFactory.builderSequential();
         // Only one column in a RangeFilter
         final Integer parquetIndex = parquetIndices.get(0);
-
-        iterateRowGroupsAndRowSet(result.maybeMatch(), (rgIdx, rs) -> {
+        final boolean[] isIdentical = {true};
```

Comment:
I think this is to allow setting the value in the lambda, but it is unusual syntax. Can you use a `MutableBoolean` instead for clarity?